src/user.java

```diff
@@ -0,0 +1,255 @@
+    protected user(){
+        firstName ="NA";
+        lastName = "NA";
+        phoneNumber = "000-000-0000";
+        email = "example@gmail.com";
+        gender = "Male";
+        birthdayYear = 0000;
+        birthdayMonth =00;
+        birthdayDay= 00;
+    }
+
+    protected user(String userFirstName, String userLastName, String userPhoneNumber, String userEmail,
+                   String userGender, int userBirthdayYear, int userBirthdayMonth, int userBirthdayDay){
+        this.firstName = userFirstName;
+        this.lastName = userLastName;
+        this.phoneNumber = userPhoneNumber;
+        this.email = userEmail;
+        this.gender = userGender;
+        this.birthdayYear = userBirthdayYear;
+        this.birthdayMonth = userBirthdayMonth;
+        this.birthdayDay = userBirthdayDay;
+
+        // * * *  * * *  * * *  * * *  * * *  * * *  * * *  * * *  * * *
+        // shouldn't these all have a 'this.' to refer to this obj? -kevin
+        // * * *  * * *  * * *  * * *  * * *  * * *  * * *  * * *  * * *
+    }
+
+
+// * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
+//              Un-used functions
+// * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
+
+/*    May need later
+public String location = "";
+public String gym = "";
+public String discipline = "";
+public String availability = "";
+public String objectives = "";
+public String routine = "";
+
+public void setLocation(String local){
+    this.location = local;
+}
+
+public void setGym(String gym){
+    this.gym = gym;
+}
+
+public void setDiscipline(String discipline){
+    this.discipline = discipline;
+}
+
+public void setAvailabilty(String avalable){
+    this.Availabilty = avalable;
+}
+public void setGoals(String goals){
+    this.goals = goals;
+}
+
+public void setRoutine(String routine){
+    this.routine = routine;
+}
+
+*/
+
+
+// * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
+//              End of Un-used functions
+// * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
+
+/*
+SET FUNCTIONS FOR ALL THE NOT COMMENTED FIELDS
+
+ */
+protected void setFirstName(String firstName){
+    if(firstName != null && firstName.length() >= 2) {
+        this.firstName = firstName;
+    }
+    else{
+        return;
+    }
+}
+
+protected void setLastName(String lastName){
+    if (lastName != null && lastName.length() >=2){
+        this.lastName = lastName;
+    }
+    else{
+        return;
+    }
+
+}
+
+protected void setPhoneNumber(String phoneNumber){
+    if (phoneNumber.matches("\\d{9}")){
+        this.phoneNumber = phoneNumber;
+    }
+    else if (phoneNumber.matches("\\d{3}[-\\s]\\d{3}[-\\s]\\d{4}"))
+    {
+        this.phoneNumber = phoneNumber;
+    }
+    else
+    {
+        return;
+    }
+}
+
+protected void setEmail(String email){
+
+    this.email = email;
+}
+
+protected void setGender(String gender){
+    if (gender == "male" || gender == "female") {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [ES: Comparison of String objects using == or != (ES_COMPARING_STRINGS_WITH_EQ)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#es-comparison-of-string-objects-using-or-es-comparing-strings-with-eq).
This code compares java.lang.String objects for reference equality using the == or != operators. Unless both strings are either constants in a source file, or have been interned using the String.intern() method, the same string value may be represented by two different String objects. Consider using the equals(Object) method instead.